torch/distributions/poisson.py

```diff
@@ -11,7 +11,9 @@ class Poisson(ExponentialFamily):
     Creates a Poisson distribution parameterized by `rate`, the rate parameter.
 
     Samples are nonnegative integers, with a pmf given by
-    $rate^k e^{-rate}/k!$
+
+    .. math::
+      rate^k e^{-rate}/k!
```

Comment:
Can you please make it `\mathrm{rate}`? Right now it renders the word as if it was a sequence of 4 variables to multiply

Reply:
Also, can you make it `\frac{e^{-\mathrm{rate}}}{k!}` to make the fraction render nicely?